sql/migrations/2023-05-29-admin_logs/up.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN TRANSACTION;
+
+CREATE TYPE bookbrainz.admin_action_type AS ENUM (
+	'Change Privileges'
+);
+
+COMMIT;
+
+BEGIN TRANSACTION;
+
+CREATE TABLE bookbrainz.admin_log (
+    id SERIAL PRIMARY KEY,
+    user1_id INT NOT NULL,
+    user2_id INT NOT NULL,
```

Comment:
The reason I am hesitating to use such naming convention is because `admin_user` sounds like a higher-ranked user in hierarchy compared to the second user. This will be true in the current use-case. But in the future, when more actions such as `Reporting a user` are added, the first user might not necessarily be higher ranked than the second one. 
It is possible that I might be reading too much into this.
For what its worth, my initial proposed schema contained similar naming. I am fine with it.

Reply:
Good point, the affected user could be an "admin" too (whatever that means in the future). Maybe `acting_user` would be a better name?

Reply:
I think we go with a simple `admin_id` and `user_id` columns. Since an 'Admin' is not an explicit user type in bookbrainz, we can say that an admin refers to anyone who is performing the administrative action, and the user is the user to which the action is directed at.
The moderation log [schema](https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/db/moderation_log.py) of CritiqueBrainz is also somewhat in the same vein, so I think this will be good enough. What do you think?

Reply:
I think `admin_id` and `target_user_id` would remove any doubt as to who is doing what, and might be a bit clearer than "actor"/"acting" or other alternatives I could think of (executor, initiator).

Reply:
How about using `user_id `for the user initiating the action and `target_id `for the user getting affected?

Reply:
I think that is more confusing at a quick glance, `user_id` could be either of the two.
Obviously `target_id` or `target_user_id` is quite clear, so perhaps `source_user_id` could also work in that case (but I still prefer `admin_id` myself.

Reply:
I am also inclined towards using `admin_id` and `target_user_id`. I think they are succinct and serve our purpose well.

Reply:
Also relevant, for what it's worth: I don't think the admin logs will show any action that hasn't been taken by an "admin". 
For example, the use-case cited above (user1 reports user2) likely won't be shown in the admin logs, but the resulting action (admin blocked user2) would be.